docs/tutorials/upgrade-a-chain/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Introduction
+---
+
+One of the defining features of the Substrate blockchain development framework is its support for
+forkless runtime upgrades. This means that Substrate-based networks can be enhanced in a way that is
+supported and protected by the capabilities of the blockchain itself. A blockchain's
+[runtime](../../knowledgebase/runtime) is the mechanism by which
+[state](../../knowledgebase/runtime/storage) changes are effected. Because the definition of the
```

Comment:
Two gross sentences. "This ____ means...", "enhanced in a way that is", "changes are effected."

Reply:
I did my best to address this comment, but next time something a little more constructive/specific would be appreciated :pray:

Reply:
It's better, I would still try to be more active. "A runtime defines the information that your blockchain can hold and the logic for effecting changes to that information. Substrate supports the capability to make breaking changes to your runtime without a hard fork." 

Then you say _how_ it supports that capability: "Because the definition of the runtime is itself an element in a Substrate chain's state..."

Reply:
"Active voice" is a writing concept that ISWTG I _cannot_ wrap my head around :sweat_smile: